### ISSUE TITLE
rpmbuild: fails because of missing pkcs11_ossl.h in the tar.gz and INSTALL.md has a typo

### ISSUE THREAD

**Issue by ceridwen** [Mastercard/pkcs11-tools#24]
After running `./configure` and `make dist`, `rpmbuild` fails:

```
  CC       libp11_la-pkcs11_ossl_fake_sign.lo
pkcs11_ossl_rsa_meth.c:28:10: fatal error: pkcs11_ossl.h: No such file or directory
   28 | #include "pkcs11_ossl.h"
      |          ^~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [Makefile:1724: libp11_la-pkcs11_ossl_rsa_meth.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
pkcs11_ossl_dsa_meth.c:28:10: fatal error: pkcs11_ossl.h: No such file or directory
   28 | #include "pkcs11_ossl.h"
      |          ^~~~~~~~~~~~~~~
compilation terminated.
pkcs11_ossl_ecdsa_meth.c:29:10: fatal error: pkcs11_ossl.h: No such file or directory
   29 | #include "pkcs11_ossl.h"
      |          ^~~~~~~~~~~~~~~
```

Once I added the missing file into the .tar.gz file generated by `make dist` and ran rpmbuild again, it succeeded and the resulting RPM installs with DNF.  This is on Fedora 32.

There's also a typo in `INSTALL.md`:

```
$ cp pkcs11-tools-[VERSION].tar.gz $HOME/rpmbuild/SRPMS
```

should be 

```
$ cp pkcs11-tools-[VERSION].tar.gz $HOME/rpmbuild/SOURCES
```

**Comment by keldonin** [comment]
Thanks a lot for reporting. I haven't gone through rebuilding RPMs yet, hence the reason why I didn't detect. Will fix that.

**Comment by keldonin** [comment]
Fixed in release v2.0.2, please give it a try and report any further issue.

Thanks,
